testDockerfile.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+EXAMPLE_COMMANDS=$(grep -e "^#   " Dockerfile | sed -e "s/^#   //"|head -2)
```

Comment:
Oops! This was a relic from my local testing!
```suggestion
EXAMPLE_COMMANDS=$(grep -e "^#   " Dockerfile | sed -e "s/^#   //")
```